src/main/java/com.bbd.bursary.manager/Repository/JdbcInstituteRepository.java

```diff
@@ -0,0 +1,31 @@
+package com.bbd.bursary.manager.Repository;
+
+import com.bbd.bursary.manager.Model.Institute;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.jdbc.core.JdbcTemplate;
+import org.springframework.stereotype.Repository;
+
+@Repository
+public class JdbcInstituteRepository implements InstituteInterface {
+    @Autowired
+    private JdbcTemplate jdbcTemplate;
+
+    public int save(Institute institute) {
+        return jdbcTemplate.update("INSERT INTO Institute_Info (InstituteName, ContactDetailsID, BBDBursaryInstituteStatus) VALUES (?,?,?)",
+                new Object[] {
+                        institute.getInstituteName()
+                });
+    }
+
+//    public int update(Institute institute) {
```

Comment:
please remove the comments

Reply:
yeah thats the one i was waiting for a response from Iviwe